src/components/Alert/Alert.module.css

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+.alert {
+  border-width: 0;
+  border-radius: 0;
+  border-left-width: 40px;
+  padding: var(--spacing-padding-lg, 16px);
+  gap: var(--spacing-gap-lg, 16px);
+  align-items: center;
+
+  background-color: var(--palette-purple-100, #EDE7FF);
+  border-color: var(--palette-purple-500, #986DF1);
+
+  & > svg {
+    position: absolute;
```

Comment:
can be done in another way?

Reply:
@Danielecina i've just made some tries but seems quite hard. the problem is that the container is a flex and hence i'm not able to set the iconContainer to be the full height of the alert with "height" or any other property (except for some kind of absolute positioning)